### ISSUE TITLE
Failed to run generate.py with rel17 and 18

### ISSUE THREAD

**Issue by Touramagan** [telekom/OpenAPI-Dissector#4]
Hi,
This is what i am encountering and was wondering if anyone has seen this error or might know what could be the root cause.

- when i run :
     PS C:\repo\GIT\OpenAPI-Dissector\generator> **py .\generate.py .\specs\**
i get:

 File "C:\repo\GIT\OpenAPI-Dissector\generator\generate.py", line 237, in <module>
    pathspec = paths[pathspec['$ref']]
               ~~~~~^^^^^^^^^^^^^^^^^^
KeyError: 'TS26510_Maf_Provisioning_ProvisioningSessions.yaml#/paths/~1provisioning-sessions~1%7bprovisioningSessionId%7d'

**Comment by Touramagan** [comment]
I noticed that this issue is related to R18 and onwards.

**Comment by lukas2511** [comment]
This should be fixed, see #3 for more information.
